app/components/content/Vid.vue

```
@@ -1,20 +1,23 @@
-</template>
 <script setup lang="ts">
-const { t } = useI18n();
-
 defineProps({
   source: { type: String },
   credit: { type: String },
   thumbnail: { type: String },
-});
+})
+
+const { t } = useI18n()
```

ESLint

[error] 'useI18n' is not defined.

(no-undef)